UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/FunctionalTests/D365HttpPayloadValidation.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Helper;
+using UKHO.ExternalNotificationService.API.FunctionalTests.Model;
+
+namespace UKHO.ExternalNotificationService.API.FunctionalTests.FunctionalTests
+{
+    class D365HttpPayloadValidation
+    {
+        private EnsApiClient _ensApiClient { get; set; }
+        private TestConfiguration _testConfig { get; set; }
+        private D365Payload _d365Payload { get; set; }
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _testConfig = new TestConfiguration();
+            _ensApiClient = new EnsApiClient(_testConfig.ensApiBaseUrl);
+
+            string filePath = Path.Combine(Directory.GetCurrentDirectory(), _testConfig.payloadFolder, _testConfig.payloadFileName);
+
```

Comment:
Rename _testConfig variables as renamed in parent class for _testConfig.ensApiBaseUrl, _testConfig.payloadFolder, _testConfig.payloadFileName.

Reply:
Done required modification.